config/rbac/role.yaml

```diff
@@ -107,6 +107,27 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - serving.kserve.io
+  resources:
+  - inferenceservices
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Are really all of them necessary? For example I don't think `create` or `delete` are necessary

Reply:
Removed